setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
Is this value only used by setuptools? If not I could see it causing unusual side-effects...